src/main/java/org/icgc/argo/program_service/converter/Grpc2JsonConverter.java

```diff
@@ -49,7 +52,30 @@ public String getJsonFromObject(Object response) {
       throw new ProgramRuntimeException(e.getMessage());
     }
   }
-
+  public UpdateProgramRequest prepareUpdateProgramRequest (UpdateProgramRequestDTO updateProgramRequestDTO) {
+
+    Program program = Program.newBuilder()
+            .addAllCancerTypes(updateProgramRequestDTO.getProgram().getCancerTypes())

```

Comment:
Not a Java dev so forgive my ignorance, but I'm seeing a lot of `updateProgramRequestDTO.getProgram()`, could this be more DRY, like captured in a variable once?

Reply:
Yes, you're right! We can capture the result of updateProgramRequestDTO.getProgram() in a variable to make the code more DRY. Will update it accordingly